doc/neps/nep-0036-fair-play.rst

```diff
@@ -100,15 +103,16 @@ Fair play rules
      '1.17.2+mycompany.15`
 
    We understand that minor patches are often required to make a
-   library work under a certain distribution.  E.g., Debian may patch
+   library work inside of a distribution.  E.g., Debian may patch
    NumPy so that it searches for optimized BLAS libraries in the
-   correct locations.  But we ask that no substantive changes are
-   made.
+   correct locations.  This is in order, but we ask that no
```

Comment:
"This is in order" sounds odd to me. Maybe just "This is fine"

Reply:
I changed it to "acceptable".